### PR TITLE
remove function remove_result_item_from_properties 🚧

### LINKED ISSUE #313: Make `remove_item_from_properties` function obsolete
This function is implemented to remove `svg` or `html` fields from GeoJSON properties before returning the API response.
Problem: This function has a high Cognitive Complexity of 25 (sonarcloud)
Solution: Do not create SVG/HTML in the first place and remove function.

This issue came up here: https://github.com/GIScience/ohsome-quality-analyst/pull/303#discussion_r857837471
> Once https://github.com/GIScience/ohsome-quality-analyst/issues/309 has been fixed, the way to include/exclude SVG in the response could be reimplemented. For example to not create the SVG in the first place (hand down the parameter to the create_figure call).
> Then this verbose function can be removed.

### PR DESCRIPTION
**Pull request by Gigaszi**
Please add the **type of change** as label. If your PR is not ready for review and merge, please add `🚧 ` to the PR title.

### Description
Make function remove_result_item_from_properties obsolet and remove it.

### Corresponding issue
Closes #313

### New or changed dependencies
- None

### Checklist
- [ ] I have updated my branch to `main` (e.g. through `git rebase main`)
- [ ] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [ ] I have commented my code
- [ ] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)
- [ ] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)
- [ ] catch requests to database

Please check all finished tasks. If some tasks do not apply to your PR, please cross their text out (by using `~...~`) and remove their checkboxes.

### CODE REVIEW

File: workers/ohsome_quality_analyst/oqt.py [hunk 1]
```diff
@@ -187,7 +203,7 @@ async def _(
             )
         )
         await db_client.save_indicator_results(indicator, dataset, feature_id)
-    indicator.create_html()
+    indicator.create_html(include_html=parameters.include_html)
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/oqt.py:206]:
```suggestion
    if parameters.include_html:
        indicator.create_html()
```

File: workers/ohsome_quality_analyst/oqt.py [hunk 2]
```diff
@@ -243,8 +259,8 @@ async def _(
     logging.info("Run calculation")
     indicator.calculate()
     logging.info("Run figure creation")
-    indicator.create_figure()
-    indicator.create_html()
+    indicator.create_figure(include_svg=parameters.include_svg)
+    indicator.create_html(include_html=parameters.include_html)
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/oqt.py:263]: See above comment

File: workers/ohsome_quality_analyst/oqt.py [hunk 3]
```diff
@@ -302,7 +318,7 @@ async def _(parameters: ReportDatabase, force: bool = False) -> Report:
         )
     report.indicators = await gather_with_semaphore(tasks)
     report.combine_indicators()
-    report.create_html()
+    report.create_html(include_html=parameters.include_html)
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/oqt.py:321]: See above comment

**Comment by Gigaszi** [comment]
In the current implementation, there is a problem that queries to the database do not work if they include or do not include the HTML or SVG. This should be discussed from our side first.

**Comment by joker234** [comment]
Outdated. `include_html` and `include_svg` will become obsolete in the future.

**Pull request closed without merging**